src/blockchain/impl/validator_registry_impl.hpp

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright Quadrivium LLC
+ * All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <unordered_map>
+
+#include <boost/di.hpp>
+#include <log/logger.hpp>
+
+#include "blockchain/validator_registry.hpp"
+
+namespace YAML {
+  class Node;
+}  // namespace YAML
```

Comment:
I think is not needed take it into namespace. Full type is more clear

Reply:
Required for forward declaration
```c++
// error: use of undeclared identifier 'YAML'
class YAML::Node;
```